frontend/src/views/Project/RolePage/components/RolePermissionsSection/ProjectRoleModifySection.utils.ts

```diff
@@ -1,29 +1,39 @@
-/* eslint-disable no-param-reassign */
```

Comment:
I'm getting this application error. This is a custom project role 

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/d626a5ca-2f53-4501-85d9-add60c7abb30">

Reply:
Can you share that JSON with me for the role permission